### DISCUSSION TITLE
Support for more complex operations?

### DISCUSSION THREAD

**Discussion by fedejinich** [tuneinsight/lattigo#398] [Q&A]
I was wondering if there's any plan to support more complex but handy operations for BFV? As I've seen there are [some studies ](https://eprint.iacr.org/2021/315.pdf) about making basic comparisons with bfv/helib.

**Comment by Pro7ech** [comment]
Such comparisons are done using polynomial evaluation, which is available in the library. You can find those polynomials with a polynomial interpolation over $\mathbb{Z}_{t}$, which is available [there](https://github.com/tuneinsight/lattigo/blob/dev_release_v4.2.0/ring/interpolation.go).

We however have no plan to implement bootstrapping for BFV or BGV at the moment.
